components/Grid.vue

```diff
@@ -1,16 +1,18 @@
 <template>
   <div ref="grid">
     <grid-view
-      :did-show-more="didShowMore"
+      :id="id"
+      :did-show-more="didShowMore(id)"
       :is-mobile="isMobile"
       :cards="cards"
       :empty-slots="emptySlots"
       :width="width"
```

Comment:
width is not used in GridView, it's only used for calculating columns here in Grid.